cvmfs/catalog_mgr_ro.h

```diff
@@ -37,13 +39,33 @@ class SimpleCatalogManager : public AbstractCatalogManager<Catalog> {
     const std::string          &dir_temp,
     download::DownloadManager  *download_manager,
     perf::Statistics           *statistics,
-    const bool                  manage_catalog_files = false)
+    const bool                  manage_catalog_files = false,
+    std::string                 dir_cache = "",
+    bool                        copy_to_tmp_dir = false)
```

Comment:
no this is the read-only catalog mgr - with logic that needs to work for both read only and writeable.
and on failure or empty cache dir it will be turned off.

for writeable catalog it is always on.